packages/vanilla/src/basics/icons/icons.scss

```diff
@@ -1,13 +1,21 @@
 @import '~helpers/css/mixins/base.scss';
 
-.hig__icon{
+.hig__icon {
     position: relative;
-    width: 30px;
-    height: 30px;
 
     & svg {
 		@include vendor-prefix(transform, translateX(-50%) translateY(-50%));
 		position: absolute;
 		top: 50%; left: 50%;
 	}
 }
+
+.hig__icon--16-size {
+  width: 20px;
```

Comment:
Best guess. This maintains the same proportion of padding-to-container as the default size.